tsfile/src/main/java/org/apache/iotdb/tsfile/encoding/decoder/DeltaBinaryDecoder.java

```diff
@@ -34,9 +28,8 @@
  * @author kangrong
  * @see DeltaBinaryEncoder
  */
-public abstract class DeltaBinaryDecoder extends Decoder {
+public abstract class DeltaBinaryDecoder<T> extends Decoder {
```

Comment:
The generic class is the primary designing and then we split it into `IntDeltaDecoder` and `LongDeltaDecoder` for efficiency. 
An efficiency evaluation is required before changing it back.

Reply:
Ok. I will revoke all these kind of modifications.